pip-requirements.txt

```diff
@@ -13,7 +13,7 @@
 ##
 
 edk2-pytool-library~=0.12.1 # MU_CHANGE - update to 0.11.0 or later
-edk2-pytool-extensions~=0.20.1 # MU_CHANGE - update to 0.17.1 or later
+edk2-pytool-extensions~=0.21.0 # MU_CHANGE - update to 0.17.1 or later

```

Comment:
If you don't mind, let's get the PRs completed in this order.

1. https://github.com/microsoft/mu_silicon_arm_tiano/pull/40
2. https://github.com/microsoft/mu_silicon_arm_tiano/pull/43
3. This one

(1) gets auto merge / auto approve in place. (2) allows dependabot to update edk2-pytool-extensions in a dedicated PR and tests the auto approve/merge process in this repo. (3) can just be your ArmGicLib changes.

Reply:
Yes, agreed. I was just messing with the pip change to see if that does anything different on the failed merge check. I will stay put on this PR till the other 2 changes get merged.